certified-connectors/iObeya/apiDefinition.swagger.json

```diff
@@ -0,0 +1,598 @@
+            "type": "integer",
+            "default": 1,
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "size",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "default": 200,
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/cards": {
+      "post": {
+        "responses": {
+          "201": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "self": {
+                  "type": "string",
+                  "description": "self"
+                },
+                "kind": {
+                  "type": "string",
+                  "description": "kind"
+                },
+                "status": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "status"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create Card",
+        "operationId": "CreateCard",
+        "parameters": [
+          {
+            "name": "Type Card",
+            "description": "Type of card",
+            "x-ms-summary": "Type Card",
+            "default": "Standard",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Standard",
+              "Activity",
+              "Story",
+              "Feature",
+              "Problem Solving",
+              "QCD Action"
+            ]
+          },
+          {
+            "name": "dynamicSchema",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "GetCardSchema",
+                "parameters": {
+                  "type": {
+                    "parameter": "Type Card"
+                  }
+                }
+              }
+            }
+          }
+        ],
+        "description": "Create a new Card",
+        "x-ms-visibility": "important"
+      }
+    },
+    "/cards/schema": {
+      "get": {
+        "summary": "Card Schema",
+        "description": "Retrieve dynamic schema of card by type",
+        "operationId": "GetCardSchema",
+        "parameters": [
+          {
+            "name": "type",
+            "type": "string",
+            "required": true,
+            "in": "query"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "ok",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/qcd/indicators-values": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default"
```

Comment:
@smiquerolle please provide schemas for all action operations. More information about response schemas [here](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses).